### ISSUE TITLE
How to set hyperparameter using the ResNet backbone (pre-trained CLIP)?

### ISSUE THREAD

**Issue by jingzhengli** [salesforce/MUST#5]
Hi, Thanks for the nice code!
On the UCF101 dataset, I used pre-trained Rsenet50 (CLIP) backbone (VIT backbone works well), but found that the performance of fine-tuning drops a lots at the beginning, about 10% compared to CLIP, and then the performance increases slowly. How shoould I set hyperparamenters such as learning rate to mitigate the drop in performance at the begining?

**Comment by LiJunnan1992** [comment]
Hi, thanks for your question, the masked image modeling loss does not apply to ResNet50 directly.

**Comment by jingzhengli** [comment]
> Hi, thanks for your question, the masked image modeling loss does not apply to ResNet50 directly.

I implement the experiments on resnet-50 through set args.mask=False.

**Comment by jingzhengli** [comment]
> Hi, thanks for your question, the masked image modeling loss does not apply to ResNet50 directly.
I only have one V100-32G; I would like to know how many GPU you use in experiments, so that I can set **the same learning rate as your experiments**.

_**total_batch_size = args.batch_size * utils.get_world_size()**_
_**args.lr = train_config['lr'] * total_batch_size / 256**_
